src/coreclr/vm/frozenobjectheap.cpp

```diff
@@ -0,0 +1,166 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "frozenobjectheap.h"
```

Comment:
```suggestion
#include "common.h"

#include "frozenobjectheap.h"
```
I think this is required to precompiled headers to kick in. We do it everywhere else in the vm